.github/workflows/generate.yml

```diff
@@ -56,25 +56,6 @@ jobs:
         uses: haskell-actions/setup@v2
         with:
           ghc-version: 9.6.4
-        # ^ ghc >= 8.2 && < 9.8 required by proto-lens-protoc 0.8.0.0
-
-      - name: Configure Cabal build
-        run: |
-          cabal configure --enable-tests --enable-benchmarks --disable-documentation
-          cabal build all --dry-run
-
-      - name: Restore Cabal build cache
```

Comment:
This step (and `Cache Cabal` below) is actually to cache the Haskell protoc plugin so that it doesn't have to be installed each time. The other actions (like `cargo-install`) automatically do this, but I couldn't find a Haskell action that automatically caches the binaries it installs.

However, I see now that the `Configure Cabal build` stage is not needed here. It wasn't actually building anything, but was used as a key for the cache, but that's not needed in this workflow.

Should we keep this step to allow for caching the plugin, or am I misunderstanding the reason this was removed?

Reply:
Hmm, looking closer, I guess I misunderstood the documentation for the Setup Haskell action and you are correct, it is caching automatically. Apologies!